app/controllers/spree/api/gift_cards_controller.rb

```diff
@@ -3,8 +3,8 @@
 module Spree
   module Api
     class GiftCardsController < Spree::Api::BaseController
-      skip_before_action :load_user, only: [:show, :update, :redeem]
-      skip_before_action :authenticate_user, only: [:show, :update, :redeem]
+      # skip_before_action :load_user, only: [:show, :update, :redeem]
```

Comment:
why comment these out instead of just removing them?

Reply:
Good catch, fixed.